### ISSUE TITLE
Add (mobile) responsiveness with tailwind

### ISSUE THREAD

**Issue by moneymanolis** [cryptoadvance/specter-desktop#2085]
In particular: 
- Sidebar collapse
- Collapsing columns in tx table

**Comment by Aditi-Singh16** [comment]
Hi! I would like to take up this issue

**Comment by moneymanolis** [comment]
@Aditi-Singh16 cool, I would suggest to familiarise yourself a bit with the relevant code, run the application etc. After we release the new UI, you could get together with @otkstdio to taggle this.

**Comment by Aditi-Singh16** [comment]
> @Aditi-Singh16 cool, I would suggest to familiarise yourself a bit with the relevant code, run the application etc. After we release the new UI, you could get together with @otkstdio to tackle this.

Sure, I have done the setup, so I assume we aren't supposed to work on any of the UIUXrevamp issues

**Comment by moneymanolis** [comment]
No, sure, go ahead, it might just take a bit of time until @otkstdio can dedicate time to review / give guidance because we are in the middle of making a release of the new UI. The responsiveness should be done with tailwind css, are you already familiar with this?

**Comment by Aditi-Singh16** [comment]
> No, sure, go ahead, it might just take a bit of time until @otkstdio can dedicate time to review / give guidance because we are in the middle of making a release with the new UI. The responsiveness should be done with tailwind css, are you already familiar with this?

Yes i have worked with tailwind 
Thanks i will give it a go
